wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -2057,11 +2179,40 @@ func OpenAICommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (sstor
 	if err != nil {
 		return nil, fmt.Errorf("openai error, cannot make dyn cmd")
 	}
+	if resolveBool(pk.Kwargs["cmdinfo"], false) {
```

Comment:
all of the cmd info / chat related functionality goes through the open ai command. I didn't add any new base commands Does that make sense? I think it does